### PR TITLE
Bug 1760501, clarified load balancing support language

### PR DESCRIPTION
**Pull request by ahardin-rh**
https://bugzilla.redhat.com/show_bug.cgi?id=1760501

### CODE REVIEW

**Comment by ahardin-rh** [comment]
@sferich888 PTAL

**Comment by ahardin-rh** [comment]
@aheslin @vikram-redhat FYI

File: install/index.adoc [hunk 1]
```diff
@@ -178,6 +178,18 @@ one HAProxy load balancer, and two
 xref:../architecture/infrastructure_components/kubernetes_infrastructure.adoc#node[nodes]
 using the `native` HA method. etcd runs as static pods on the master nodes:
 
+[IMPORTANT]
+====
+Routers and master nodes must be load balanced to have a highly available and
+fault-tolerant environment. Red Hat recommends the use of an enterprise-grade
+external load balancer for production environments. This Load balancing applies
+to the masters, nodes, and hosts running the {product-title} routers.
```
Comment (sferich888) [inline install/index.adoc]: nodes (hosts running the {product-title} routers.

File: install/index.adoc [hunk 2]
```diff
@@ -178,6 +178,18 @@ one HAProxy load balancer, and two
...
+Transmission Control Protocol (TCP) layer 3/4 load balancing, in which the load
```
Comment (sferich888) [inline install/index.adoc]: We may just want to say layer 4 

@sheriff-rh can you comment here if layer 3/4 is correct (my networking, terminology is rusty).

File: install/index.adoc [hunk 3]
```diff
@@ -178,6 +178,18 @@ one HAProxy load balancer, and two
...
+is spread across IP addresses, is recommended. See
+link:http://v1.uncontained.io/playbooks/installation/load_balancing.html[External
+Load Balancer Integrations with OpenShift Enterprise 3] for a reference design.
```
Comment (sferich888) [inline install/index.adoc]: We may want a note with this that (we don't recommend this 'example' for production use-cases; as it itself is a single point of failure (SOPF), and it's only when you pair this with information from: https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/7/html-single/load_balancer_administration/index#s2-lvs-keepalived-haproxy-VSA can you get an HA deployment of Haproxy (from Red Hat) that creates a robust enterprise grade load balancer.

**Review by sferich888 — APPROVED** [review]

**Comment by ahardin-rh** [comment]
@sferich888 Thanks! Updated. @sheriff-rh PTAL at the language from a networking perspective.

File: install/index.adoc (hunk 2 above)
Comment (sheriff-rh) [reply install/index.adoc]: Correct @sferich888 TCP is layer 4.

**Comment by ahardin-rh** [comment]
@sferich888 Updated. Thanks!
@xltian I'm not sure if this needs QE review, but if it does, can you please assign someone? Thank you!

**Comment by sheriff-rh** [comment]
LGTM @ahardin-rh !

**Comment by lihongan** [comment]
LGTM, thanks

**Comment by jianlinliu** [comment]
LGTM.

**Comment by ahardin-rh** [comment]
Thank you!

**Pull request merged**